spec/rspec/core/option_parser_spec.rb

```diff
@@ -394,5 +394,24 @@ module RSpec::Core
       end
     end
 
+    describe '--force-color' do
+      it 'aborts if --no-color was previously set' do
+        parser = Parser.new(%w[--no-color --force-color])
+        expect(parser).to receive(:abort) do |msg|
+          expect(msg).to include('only use one of --force-color and --no-color')
+        end
```

Comment:
I think this can be simplified to:

``` ruby
expect(parser).to receive(:abort).with(a_string_including("only use one of --force-color and --no-color"))
```

No need for the nested expectation inside the block :).